controllers/admin/AdminOrdersController.php

```diff
@@ -478,19 +478,64 @@ public function postProcess()
                         // Send mail to customer
                         $customer = new Customer((int)$order->id_customer);
                         $carrier = new Carrier((int)$order->id_carrier, $order->id_lang);
+                        $address = new Address((int)$order->id_address_delivery);
+
                         if (!Validate::isLoadedObject($customer)) {
                             throw new PrestaShopException('Can\'t load Customer object');
                         }
                         if (!Validate::isLoadedObject($carrier)) {
                             throw new PrestaShopException('Can\'t load Carrier object');
                         }
+                        if (!Validate::isLoadedObject($address)) {
+                            throw new PrestaShopException('Can\'t load Address object');
+                        }
+
+                        $products = $order->getCartProducts();
+
+                        $metadata = '';
+
+                        $link = Context::getContext()->link;
+
+                        foreach ($products as $product) {
+                            $metadata .= '<div itemprop="itemShipped" itemscope itemtype="http://schema.org/Product">';
```

Comment:
@inetbiz It would not at all beacause this controller only manages the email that is sent when the tracking number is updated via the Back Office. 
For other kind of emails, other controllers have to be modified, with the adequate schema.org markup, so there is no problem.